### PR TITLE
Mitigate GUI DoS (part 2: qvm-xkill)

### PR DESCRIPTION
**Pull request by jpouellet**
Can close windows of a VM while it's paused, and can not accidentally
harm dom0 by errant clicking.

Discussion in https://github.com/QubesOS/qubes-issues/issues/881

Thanks to rustybird for suggested implementation.

### CODE REVIEW

**Comment by jpouellet** [comment]
If we agree this is the right direction, next steps are:
- Replacing current Ctrl+Alt+Escape -> xkill shortcut with qvm-xkill.
- Documentation updates.

File: qvm-xkill [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+ID=$(xdotool selectwindow)
+
+xprop -id "$ID" _QUBES_VMNAME | grep -q ' = ' \
+  || { echo "${0##* /}: Not killing dom0 window $ID" >&2; exit 1; }
```
Comment (rustybird) [inline qvm-xkill]: Should be `${0##*/}` (without the space), other than that LGTM.

Comment (jpouellet) [inline qvm-xkill]: Ah yep. Good catch. Thanks!

**Comment by jpouellet** [comment]
Just forced pushed to avoid useless 2nd pre-merge commit.

**Comment by jpouellet** [comment]
Travis build error not due to this change:
```
+ yumdownloader --resolve --destdir=/home/travis/qubes-builder/cache/fc23/base_rpms -c /tmp/tmp.7gJ1Gu82pX -y --installroot=/home/travis/qubes-builder/chroot-fc23 filesystem setup fedora-release dnf dnf-plugins-core
Cannot retrieve repository metadata (repomd.xml) for repository: fedora. Please verify its path and try again
make[1]: *** [/home/travis/qubes-builder/chroot-fc23/home/user/.prepared_base] Error 1
make[1]: Leaving directory `/home/travis/qubes-builder'
make: *** [core-admin-linux-dom0] Error 1
```

**Comment by marmarek** [comment]
Please, move this to some directory, don't create new top-level files if not necessary. I don't see any directory where this should belong (any idea?), so create new one, like `tools` or such.

**Comment by jpouellet** [comment]
> Please, move this to some directory, don't create new top-level files if not necessary. I don't see any directory where this should belong (any idea?), so create new one, like tools or such.

Done.

**Pull request merged**